web/pdf_page_view.js

```diff
@@ -494,7 +496,12 @@ class PDFPageView {
     }
   }
 
-  update({ scale = 0, rotation = null, optionalContentConfigPromise = null }) {
+  update({
+    scale = 0,
+    rotation = null,
+    optionalContentConfigPromise = null,
+    postponeDrawingDelay = -1,
```

Comment:
As mentioned previously, for overall consistency:
```suggestion
    drawingDelay = -1,
```